protocol/messenger.go

```diff
@@ -126,13 +126,6 @@ type config struct {
 
 type Option func(*config) error
 
-func WithSystemMessagesTranslations(t map[protobuf.MembershipUpdateEvent_EventType]string) Option {
```

Comment:
This is currently not used, but will be used once we enable group chats, but we can remove it for the time being and re-introduce it, no worries